indica/readers/abstractreader.py

```diff
@@ -15,20 +15,24 @@
 from indica.converters.line_of_sight import LineOfSightTransform
 from indica.converters.transect import TransectCoordinates
 from indica.datatypes import ArrayType
-from indica.numpy_typing import OnlyArray
 from indica.numpy_typing import RevisionLike
 from indica.readers.available_quantities import AVAILABLE_QUANTITIES
 from indica.utilities import format_dataarray
 
 
 def instatiate_line_of_sight(
-    location: OnlyArray,
-    direction: OnlyArray,
+    database_results: dict,
```

Comment:
Is there a reason for doing this? I don't like hiding a functions args in a dictionary; it makes it less clear what a function is doing.

Reply:
I don't think it makes sense to use up 3 lines in every reader method when you can have 1. 
We could also move it to a private method to make it even clearer?

Reply:
Maybe a compromise option, could use the "**" expansion for function args? So keep the named arguments being the same as the keys in `database_results` and in the calls do `instantiate_line_of_sight(**database_results)` ? Would just need an extra `*args` and `**kwargs` in the `instantiate_line_of_sight` arguments to catch the extra values we're not interested in, then should work?